packages/ui-server/routes/api/_tooltip.js

```diff
@@ -0,0 +1,50 @@
+/**
+ *
+ * Tooltip object that consists of tooltips for various of
+ * features
+ *
+ * @returns {Object} An Object that consists of tooltip methods to be invoked
+ */
+module.exports = {
+	cssPlugin: () => `/*
+* We've enabled MiniCssExtractPlugin for you. This allows your app to
+* use css modules that will be moved into a separate CSS file instead of inside
+* one of your module entries!
+*
+* https://github.com/webpack-contrib/mini-css-extract-plugin
+*
+*/`,
+	splitChunks: () => `/*
+* SplitChunksPlugin is enabled by default and replaced
+* deprecated CommonsChunkPlugin. It automatically identifies modules which
+* should be splitted of chunk by heuristics using module duplication count and
+* module category (i. e. node_modules). And splits the chunks…
+*
+* It is safe to remove "splitChunks" from the generated configuration
+* and was added as an educational example.
+*
+* https://webpack.js.org/plugins/split-chunks-plugin/
+*
+*/`,
+	postcss: () => `/*
+* We've enabled Postcss, autoprefixer and precss for you. This allows your app
+* to lint  CSS, support variables and mixins, transpile future CSS syntax,
+* inline images, and more!
+*
+* To enable SASS or LESS, add the respective loaders to module.rules
+*
+* https://github.com/postcss/postcss
+*
+* https://github.com/postcss/autoprefixer
+*
+* https://github.com/jonathantneal/precss
+*
+*/`,
+	uglify: () => `/*
```

Comment:
We're using terser

Reply:
Yeah, will do it here when terser PR get merged in master 👍